js/logic.js

```diff
@@ -345,6 +361,39 @@ const logic = (function () {
     return minutes + ':' + seconds;
   }
 
+  /**
+   * Displays Flow state on timer
+   * @return {boolean}
+   */
+  function displayFlowTime() {
+    if (isFlowing) {
+      chatHandler.chatItalicMessage(responses.flowingMsg);
+      return false;
+    }
+    if (isRunning && !isPaused) {
+      chatHandler.chatItalicMessage(responses.timerRunning);
+      return false;
+    }
```

Comment:
Does this mean if it's already running, e.g. with `!timer start` that means that the user will not be able to switch to flow mode?